tests/test_plugin.py

```diff
@@ -30,21 +32,26 @@ def test_adding_widget_to_viewer(viewer_widget):
 def test_basic_function(qtbot, viewer_widget):
     viewer, widget = viewer_widget
     viewer.open_sample('cellpose-napari', 'rgb_2D.png')
+    viewer.layers[0].data = viewer.layers[0].data[0:128, 0:128]
 
-    if os.getenv("CI"):
-        return
+    #if os.getenv("CI"):
+    #    return
         # actually running cellpose like this takes too long and always timesout on CI
         # need to figure out better strategy
-    widget.compute_diameter_button.changed(None)
+    #widget.compute_diameter_button.changed(None)
```

Comment:
this is the only line I'm curious about.  don't know what this was doing... but why commenting it out?

Reply:
Oh, sorry. Yeah that was computing the diameter from the image rather than using the default. I wanted this test to be as barebones as possible.
How about I add a new test just for that function?

Reply:
if it's not getting tested anywhere else, then yeah, a quick new test just to make sure we're not losing coverage anywhere would be good, thanks

Reply:
Added a test for simple case, compute diameter from image, which was what was being used that I commented out to make tests quicker.
I've never used the other function that uses a shapes layer...

Reply:
Sigh
Ubuntu is stalling?
😩